Cineaste.xcodeproj/project.pbxproj

```diff
@@ -105,9 +105,7 @@
 		3FF7911F20DE274A007B7D37 /* Date+Formatter.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3FBD8F911FF3FC5B00AD5F02 /* Date+Formatter.swift */; };
 		3FF7912020DE274A007B7D37 /* Webservice.swift in Sources */ = {isa = PBXBuildFile; fileRef = 95EFA4AD1FA13878004F1F94 /* Webservice.swift */; };
 		3FF7912120DE274A007B7D37 /* ShortcutIdentifier.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3F98063920554AE300B92A27 /* ShortcutIdentifier.swift */; };
-		3FF7912520DE274A007B7D37 /* .swiftlint.yml in Resources */ = {isa = PBXBuildFile; fileRef = 9528112F201D087800F3EC76 /* .swiftlint.yml */; };
 		3FF7912620DE274A007B7D37 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 95AE25A81F9531E20067F5F5 /* LaunchScreen.storyboard */; };
-		3FF7912720DE274A007B7D37 /* cineaste.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 958328991FBB81340084CF80 /* cineaste.xcassets */; };
```

Comment:
Seems like this change is responsible for not having an app icon at all when running the app (at least for the simulator). Are you sure we can remove the assets here? Would be glad if the warning would go away, but I somehow thought it is a bug, because there are no duplicated resources in the Copy Bundle Resources.

Reply:
You're right! I'll investigate some more...

Reply:
I found the offending file:

<img width="973" alt="screenshot 2018-10-31 at 15 14 23" src="https://user-images.githubusercontent.com/16212751/47794015-dd389e00-dd1f-11e8-8eb1-b82fa3e8d865.png">

Google Nearby has two `.xcasset` files which are apparently also compiled to an `Assets.car`.

If you delete the line from the output files list, the warning disappears.
However, I do not believe that this is safe. I tried removing all Cocoapods specific build phases, `pod deintegrate` and `pod install`, and the output file appears again.

Reply:
Oh.. I see. Yeah, i tried that as well... Seems like this warning is appearing since a pod update from nearby. Thanks for your investigation. If you undo the changes of removing the assets in your PR again, I can merge this. 😃

Reply:
Too bad, I really wanted to conquer that warning...